resources/lang/en/events/lead_ceremony_df.json

```
@@ -128,7 +128,572 @@
+        }
+      ]
+    },
+  
+    "faithful_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["faithful"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "r_c's eyes glitter with bleak conviction as {PRONOUN/r_c/subject} {VERB/r_c/approach/approaches} m_c. Offering a life for [virtue], r_c murmurs that m_c must follow {PRONOUN/m_c/poss} beliefs - no matter what path those beliefs may lead {PRONOUN/m_c/object} down."
+        }
+      ]
+    },
+  
+    "fierce_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["fierce"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "Only the strongest could survive being leader according to r_c's ferocious growl. {PRONOUN/r_c/subject/CAP} {VERB/r_c/press/presses} {PRONOUN/r_c/poss} claws against m_c's neck while giving a life for [virtue], encouraging the new leader to become a story to haunt the minds of the other Clan's kits.",
+          "virtue": ["ambition", "bravery", "certainty", "clear judgment", "confidence", "courage",
+      "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunning", "shamelessness", "zealotry", "bloodlust", "restless energy", 
+      "stubbornness", "authority"]
+        },
+        {
+          "text": "r_c's claws dug into the shadowy ooze underneath {PRONOUN/r_c/object} when {PRONOUN/r_c/subject} {VERB/r_c/approach/approaches} m_c. The dark spirit gives a life for [virtue]. It feels like the claws of all the foxes in the world are raking down m_c's pelt.",
+          "virtue": ["ambition", "bravery", "certainty", "clear judgment", "confidence", "courage",
+      "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunning", "shamelessness", "zealotry", "bloodlust", "restless energy", 
+      "stubbornness", "authority"]
+        }
+      ]
+    },
+  
+    "insecure_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["insecure"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "r_c darts a nervous look over {PRONOUN/r_c/poss} shoulder as {PRONOUN/r_c/subject} {VERB/r_c/approach/approaches}, studying the gloom for one skittish moment. {PRONOUN/r_c/subject/CAP} {VERB/r_c/turn/turns} back to m_c and {VERB/r_c/thrust/thrusts} {PRONOUN/r_c/poss} muzzle against m_c's hard enough to bruise, forcing a life for [virtue] on {PRONOUN/m_c/object} before withdrawing.",
```

```suggestion
          "text": "r_c approaches, shooting a nervous look over {PRONOUN/r_c/poss} shoulder to study the gloom for one fleeting moment. {PRONOUN/r_c/subject/CAP} {VERB/r_c/turn/turns} back to m_c and {VERB/r_c/thrust/thrusts} {PRONOUN/r_c/poss} muzzle against {PRONOUN/m_c/inposs}, hard enough to bruise. r_c forces a life for [virtue] onto {PRONOUN/m_c/object} before withdrawing.",
```
May have overworked this one too. Feel free to take what sparks joy and what doesn't.